Release notes 2026.07.23 (#346)
organicmaps/organicmaps.github.io: content/news/2026-07-23/620/index.md

- Track elevation profiles no longer show outdated or empty charts after a track is deleted _(Kiryl Kaveryn)_

### Public transport
- Stop, transfer, and station names now have a white outline to stay readable in both light and dark themes _(Viktor Govako)_
- The subway layer reappears correctly after you close a public transport route preview _(Mikhail Listratsenka)_

### Routing & navigation
- Route warnings (tolls, ferries, unpaved roads, steps, and so on) are now shown for all alternate routes _(Viktor Govako)_
- Fixed a rare freeze while building a route _(Viktor Govako)_
- Improved handling of dead ends and of start and finish points on restricted roads _(Viktor Govako)_
- Fixed incorrect and missing turn instructions _(Alexander Borsuk)_

### iOS
- New "Save search history" setting that lets you turn the history off and hide it if you would rather not keep it _(Kiryl Kaveryn)_
- New Edit button to remove bookmarks more easily _(Kiryl Kaveryn)_
- Bookmarks are now saved automatically when you leave the screen _(Kiryl Kaveryn)_
- The color palette now offers predefined colors and lets you choose any custom color _(Kiryl Kaveryn)_

> - Improved the empty elevation chart for a recorded track _(Kiryl Kaveryn)_

Author: "Improved the empty elevation chart" reads as improving a chart that stays empty. Presumably this is about the empty-state placeholder shown when a recorded track has no elevation data. All 32 translations copied the ambiguity verbatim (only `de` and `nl` added "the display of"), so clarifying the source fixes them all:

```suggestion
- Improved the empty state of the elevation chart for a recorded track _(Kiryl Kaveryn)_
```

---
_Generated by [Claude Code](https://claude.ai/code)_

- Improved the route progress shown on the Start button _(Kiryl Kaveryn)_
- Reordering route stops no longer makes the list jump around _(Kiryl Kaveryn)_
- Other minor interface improvements _(Kiryl Kaveryn)_

### Android
- Opening hours now show split shifts, such as a lunch break, start from today, and display the whole week without a separate scrolling area _(Owm Dubey, Alexander Borsuk, Mikhail Listratsenka)_

> - Cleaner search bar with a combined clear and voice button, a clear icon that no longer moves, and fixed layouts in landscape mode and after you rotate the phone _(Mikhail Listratsenka)_

Author: `fixed layouts` is ambiguous (past tense "repaired" vs. adjective "immovable"), and it has already broken 26 of the 32 translations, which read it as *static/immovable layouts*:

`af` vaste uitlegte · `ar` تصميمات ثابتة · `cs` pevným rozložením · `cy` chynlluniau sefydlog · `de` festen Layouts · `el` σταθερές διατάξεις · `es` diseños fijos · `et` fikseeritud paigutus · `eu` finkatutako diseinuekin · `fa-IR` چیدمان‌های ثابت · `fr` mises en page stabilisées · `gl` deseños fixos · `he` פריסות קבועות · `id` tata letak yang tetap stabil · `it` layout fissi · `ml` സ്ഥിരമായ ലേഔട്ടുകൾ · `mr` निश्चित केलेले लेआउट · `nl` vaste lay-outs · `oc` disposicions fixas · `pl` ustalonymi układami · `pt` / `pt-BR` layouts fixos · `sv` fasta layouter · `te` స్థిరంగా ఉండే లేఅవుట్‌లు · `tr` sabit kalan düzenler · `uk` фіксованими макетами

Only `ca`, `hi`, `hu`, `lt`, `ru`, `zh-Hans` got it right.

Suggested unambiguous source wording, then re-sync the affected translations:

```suggestion
- Cleaner search bar with a combined clear and voice button, a clear icon that no longer moves, and layout fixes for landscape mode and phone rotation _(Mikhail Listratsenka)_
```

---
_Generated by [Claude Code](https://claude.ai/code)_
